src/main/java/com/client/core/base/util/Utility.java

```diff
@@ -20,23 +21,13 @@
 import org.joda.time.DateTimeZone;
 import org.joda.time.Days;
 import org.json.JSONArray;
-import org.springframework.beans.factory.annotation.Autowired;
 
-import javax.xml.datatype.DatatypeConfigurationException;
-import javax.xml.datatype.DatatypeFactory;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Date;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
Usually standard to try to not import all of java util packages and only the ones we need

Reply:
This is my IDE automatically doing this. I'll disable the option.